pages/dkp/kommander/1.3/release-notes/index.md

```diff
@@ -11,59 +11,42 @@ enterprise: false
+
 # Supported Versions
 | Kubernetes Support | Version |
 | ------------------ | ------- |
 | **Minimum**        | 1.16.0  |
 | **Maximum**        | 1.18.x  |
 | **Default**        | 1.18.8  |
-# New Features and Capabilities 
-## Air gapped environments
-Kommander now supports installing and running in an air gapped environment, either on premises or in the cloud. Air gapped environments are ideal when high-security is a must or when Internet connectivity is undesirable or unavailable. For information on setting up Kommander in an air gapped environment, see [Install Kommander air gapped](/ksphere/kommander/1.3/install-airgapped/).
+# New Features and Capabilities
+
 ## Component versions
-- Addon: 1.2.0-36
-- Chart: 0.12.16
-- kommander-federation (yakcl): 0.6.9
-- kommander-licensing (yakcl): 0.6.9
-- UI: 6.37.0
-- kommander-karma: 0.3.10
+- Addon: 1.3.0-6
+- Chart: 0.13.7
+- kommander-federation (yakcl): 0.7.0
+- kommander-licensing (yakcl): 0.7.0
+- UI: 6.70.0
+- kommander-karma: 0.3.12
 - kubeaddons-catalog: 0.1.15
-- kommander-thanos: 0.1.15
+- kommander-thanos: 0.1.16
 - kubecost: 0.1.15
 - grafana: 6.6.0
 - karma: 0.70
 - thanos: 0.10.1
-- cost-analyzer: 1.67.1
+- cost-analyzer: 1.70.1
 
-## Fixed and Improved Issues 
-- Improved Access Control for editing/deleting actions. 
-- Changed routing from hash-based to history-based to allow redirects after login.
-- Added SAML IDP.
-- Moved provisioning code to Konvoy. Kommander now uses Konvoy to provision clusters.
-- Added a setting to configure thresholds for resource warnings. 
-- Added the ability to show metrics in attached EKS clusters.
-- Added Infrastructure provider management on global level.
-- Fixed an issue where logout was not deleting any cookies.
-- Fixed UI crashes when kubecost was down.
-- Fixed an issue where the catalog API occasionally returned 500.
-- Fixed an issue that prevented deleting cluster labels.
-- Fixed an issue that erroneously counted the self-attached host cluster against the license count.
-- Updated cert-manager resources to v1.
-- Added automatic creation of **KommanderProjectRole** and **VirtualGroupKommanderProjectRoleBinding** objects based on Workspace objects.
-- Fixed access to kubecost UI for some cluster roles.
-- Created the serviceaccount needed for deletion jobs on delete.
-- Added --ignore-not-found to resource cleanup jobs to ignore errors if resource has already been deleted.
-- Fixed metric collection for kubefed, kommander-federation, and kommander-licensing.
+## Fixed and Improved Issues
+- Added Kubecost Prometheus health dashboards to Grafana.
+- Added the ability to edit network policies.
```

Comment:
Looks probably belongs under `New Features and Capabilities` https://github.com/mesosphere/dcos-docs-site/pull/3375/files#diff-25401265c1b8ce2a4f0f4e8bb9d876aad31221c73a6c9f0a3de2bf9193000697R31

Reply:
I was mainly following the previous pattern, there was a longer airgapped section under the "new features" heading, but all other bullet points were put under "fixed and improved". I'll defer to @cmastr 
https://docs.d2iq.com/dkp/kommander/1.2/release-notes/#new-features-and-capabilities

Reply:
So @samvantran  is right that these are features. But as we discussed via Slack, the changes to the related topics has not been completed yet, so there will be not place to link them to for now. As I said, for now I'm ok with these being under the improvements and fixes section since it's so early in the release cycle, but we will need to clean this up before GA.

Reply:
sgtm